frontend/src/components/tutorialRequestModal/TutorialRequestModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { Form, Modal, Input, Button, message, Select } from 'antd';
+import TextArea from "antd/lib/input/TextArea";
+import { useForm } from "antd/lib/form/Form";
+import { useState } from "react";
+import { createTutorialRequest } from "../../api/api";
+import { TutorialRequest } from "../../types/Tutorial";
+import { validateMessages } from "../../utils/Messages";
+
+interface Props {
+    isModalVisible: boolean,
+    setIsTutorialRequestModalVisible: (visible: boolean) => void
+}
+
+export const TutorialRequestModal: React.FC<Props> = ({ isModalVisible, setIsTutorialRequestModalVisible }) => {
+
+    const [loading, setLoading] = useState(false);
+    const [form] = useForm();
+    const { Option } = Select;
+
+    const onFinish = (values: any) => {
+        console.log(values.semester);
+        setLoading(true);
+        createTutorialRequest({
+            title: values.title,
+            semester: Number(values.semester),
+            description: values.description
+        } as TutorialRequest).then(res => {
+            setLoading(false);
+            message.success("Tutoriumsanfrage erfolgreich erstellt.");
+            setIsTutorialRequestModalVisible(false);
+            form.resetFields()
+        }, err => {
+            setLoading(false);
+            message.error("Tutoriumsanfrage konnte nicht erstellt werden.");
+        }
+
+        )
+    };
+
+    const onCancel = () => {
+        setIsTutorialRequestModalVisible(false)
+        form.resetFields()
+    }
+
+    return (
+        <Modal
+            title="Tutoriumsanfrage erstellen"
+            destroyOnClose={true}
+            visible={isModalVisible}
+            onCancel={onCancel}
+            width={600}
+            footer={[
+                <Button loading={loading} htmlType="submit" type="primary" onClick={e => form.submit()}>
+                    Absenden
+                </Button>
+            ]}>
+            <Form
+                onFinish={onFinish}
+                form={form}
+                labelCol={{ span: 5 }}
+                wrapperCol={{ span: 17 }}
+                validateMessages={validateMessages}
+            >
+                <Form.Item label="Titel" name="title" rules={[{ required: true },{whitespace: true}]}>
+                    <Input placeholder="Programmieren I" allowClear />
+                </Form.Item>
+                <Form.Item label="Dein Semester" name="semester" rules={[{required:true}]}>
+                    <Select style={{ width: 60 }} allowClear>
+                        <Option value="1">1</Option>
+                        <Option value="2">2</Option>
+                        <Option value="3">3</Option>
+                        <Option value="4">4</Option>
+                        <Option value="5">5</Option>
+                        <Option value="6">6</Option>
+                    </Select>
```

Comment:
Hier wäre dann noch ein Select für die Studienrichtung nötig. Da es definierte Werte vom Backend gibt wäre dazu folgendes nötig:
1. Backend: Route definieren, die mögliche `SpecialisationCourses` zurückgibt.
2. Frontend: Im types-Ordner ein Interface für SpecialisationCourse erstellen. In `api.tsx` die Route aufrufen und die Ergebnisse zum Typ mappen.
3. Frontend: beim laden des Modals mithilfe von `useEffect` die Studienrichtungen abrufen und das Ergebnis in ein neues `<Select>` packen, aus dem der Nutzer dann wählen kann
4. 
Melde dich gerne wenn du Hilfe brauchst.

Reply:
Backend route in specialisationCourse in einem neuen Controlleer. Da dann die Route definieren und alles

Reply:
Die Auswahl der Spezialisierung im Frontend wurde verworfen, somit ist das hier resolved.